static/app/views/settings/account/accountNotificationFineTuning.tsx

```diff
@@ -128,7 +129,8 @@ type State = AsyncView['state'] & {
 
 class AccountNotificationFineTuning extends AsyncView<Props, State> {
   getEndpoints(): ReturnType<AsyncView['getEndpoints']> {
-    const {fineTuneType} = this.props.params;
+    const {fineTuneType: param} = this.props.params;
```

Comment:
Instead of just `param` you can probably call the variable something like `pathnameType` so that it's clear it isn't the NotificationType anymore.